index.html

```diff
@@ -163,6 +163,32 @@ <h3 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h3>
                   </li>
               </ul>
              </section>
+
+             <section class="form">
+                <form action="https://formspree.io/f/mnqygqol" method="post">
+                <h2 class="form-header">
+                    Contact me
+                </h2>
+                <p>
+                    If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!
+                </p>
+                <ul class="ul_form">
+                    <li>
+                        <input type="text" maxlength="30" name="name" placeholder="enter your name......" required>
+                    </li>
+                    <li>
+                        <input type="text" name="email" placeholder="enter your email......"required>
```

Comment:
[OPTIONAL]

- Kindly ensure that your email input fields have the correct type attribute i.e email input's `type` equals `email` which is the proper way of declaring an input field 👍🏽